code/game/turfs/open/floor.dm

```diff
@@ -161,14 +161,19 @@
 	W.update_icon()
 	return W
 
-/turf/open/floor/attackby(obj/item/C, mob/user, params)
-	if(!C || !user)
-		return 1
+/turf/open/floor/attackby(obj/item/object, mob/user, params)
+	if(!object || !user)
+		return TRUE
 	if(..())
-		return 1
-	if(intact && istype(C, /obj/item/stack/tile))
-		try_replace_tile(C, user, params)
-	return 0
+		return TRUE
+	if(intact && istype(object, /obj/item/stack/tile))
+		try_replace_tile(object, user, params)
+		return FALSE
+	if(istype(object, /obj/item/stack/sheet))
+		var/obj/item/stack/sheet/sheets = object
+		sheets.attacked_by_floor(user, params)
+		return FALSE
```

Comment:
Personally I think that returning the `attacked_by_floor` return is the best
That said, are you sure you want to return `FALSE` when the attack chain should be successful here?

Reply:
I see what you are saying, i will rework this.